core/src/types/governance.rs

```diff
@@ -13,7 +13,7 @@ use crate::types::hash::Hash;
 use crate::types::key::common::{self, Signature};
 use crate::types::key::SigScheme;
 use crate::types::storage::Epoch;
-use crate::types::token::{Amount, SCALE};
+use crate::types::token::{Amount, NATIVE_SCALE};
 
 /// Type alias for vote power
 pub type VotePower = u128;
```

Comment:
Given what we said in the meeting, this `u128` is no longer needed. I'd still keep the type alias as
```rust
pub type VotePower = Uint;
```
to use a more appropriate term for this context. We could probably just use an `Amount` but this is something that we can refactor later on with @Fraccaman.